reactor-netty-core/src/main/java/reactor/netty/resources/ConnectionProvider.java

```diff
@@ -175,7 +175,12 @@ static ConnectionProvider create(String name, int maxConnections) {
 			@Nullable AddressResolverGroup<?> resolverGroup);
 
 
-	default void disposeWhen(SocketAddress address) {
+	/**
+	 * Dispose all connection pools for the specified remote address.
```

Comment:
maybe mention the default implementation is NO-OP but should be implemented in user-provided implementations the same way `disposeLater` should be implemented?

Reply:
done